missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/PlayerListener.java

```diff
@@ -209,6 +211,16 @@ private boolean checkJoinOrLeave(Player player, Game from, Game to) {
         return false;
     }
 
+    // cancel anvil interact to block the way to rename
+    // missile spawn-egg for changing the spawn
+    @EventHandler
+    public void onClick(InventoryClickEvent event) {
```

Comment:
This event would also be cancelled in non MissileWars parts on the server. Please make it Game Specific and check if it is in the GameWorld